### PR TITLE
Create a database to store information

### PR DESCRIPTION
**Pull request by elineelorm**
Here are the changes I made

- Added the formhandler 
- Created the database to store the form handler information (I deployed my changes so they can be seen)
- Updated the CSS to make the form look nicer.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java [hunk 1]
```diff
@@ -9,16 +14,25 @@
 @WebServlet("/form-handler")
 public class FormHandlerServlet extends HttpServlet {
 
-  private static final String REDIRECT_URL = "https://enuviadenu-sps-summer21.uc.r.appspot.com/index.html";
+  private static final String REDIRECT_URL = "https://enuviadenu-sps-summer21.uc.r.appspot.com/contact.html";
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java]: formatting nit: make sure the lines are 80chars. Move the "http://..." to the next line

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java [hunk 2]
```diff
@@ -9,16 +14,25 @@
...
 
-    System.out.println("You submitted: " + textValue);
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    KeyFactory keyFactory = datastore.newKeyFactory().setKind("Task");
+    FullEntity textEntity =
+        Entity.newBuilder(keyFactory.newKey())
+            .set("name", name)
+            .set("text-input", textValue)
+            .build();
+    datastore.put(textEntity);
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java:41]: You could move this block to a writeToDatastore or saveToDatastore method.

**Review by sbahaud — COMMENTED** [review]
Added a few comments

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java (hunk 1 above)
Comment (elineelorm) [reply portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java]: fixed

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java (hunk 2 above)
Comment (elineelorm) [reply portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java:41]: done

**Review by sbahaud — APPROVED** [review]

**Pull request merged**